ci/run_cudf_polars_pytests.sh

```diff
@@ -8,4 +8,8 @@ set -euo pipefail
 # Support invoking run_cudf_polars_pytests.sh outside the script directory
 cd "$(dirname "$(realpath "${BASH_SOURCE[0]}")")"/../python/cudf_polars/
 
+# Test the default "cudf" executor
 python -m pytest --cache-clear "$@" tests
+
+# Test the "dask-experimental" executor
+python -m pytest --cache-clear "$@" tests --executor dask-experimental
```

Comment:
Are we guaranteed to have dask installed when CI runs?

Reply:
IIUC, https://github.com/rapidsai/cudf/pull/17062/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6R752-R756 added `rapids-dask-dependency` to the experimental requirements, so my understanding is that we are guaranteed to have Dask installed.

Reply:
Yes